test/udp_sender.js

```diff
@@ -108,6 +110,12 @@ describe('udp sender should', () => {
         server.on('message', (msg, remote) => {
             let thriftObj = thrift.Agent.emitBatch.argumentsMessageRW.readFrom(msg, 0);
             let batch = thriftObj.value.body.batch;
+            let span = batch.spans[0];
+            let ref = span.references[0];
+
+            assert.isOk(bufferEqual(span.traceIdLow, ref.traceIdLow));
```

Comment:
unclear what the business meaning of this check is. I'd rather expect a check that the span's traceID/parentID came from childOf reference.